tsconfig.json

```diff
@@ -1,12 +1,11 @@
 {
+  "extends": "@tsconfig/node14/tsconfig.json",
```

Comment:
If some of the share config should be used. 
I would go for `fastify-tsconfig` which we have control.

Although, it is a shared config provided by `TypeScript`. 
I would still avoid using something that we do not control since it will affect the compiled code.

Reply:
Fastify-tsconfig seemed dead. I dont even think that we can easily agree on the basic tsconfig.

Reply:
I don't think it's dead. I don't see a reason to not use it here, it's customizable.